newIDE/app/src/UI/KeyboardShortcuts/index.js

```diff
@@ -125,17 +126,21 @@ export default class KeyboardShortcuts {
     if (this._isControlPressed() && evt.which === Y_KEY) {
       this.onRedo();
     }
-    if (this._isControlPressed() && evt.which === MINUS_KEY) {
-      this.onZoomOut();
-    }
-    if (this._isControlPressed() && evt.which === EQUAL_KEY) {
-      this.onZoomIn();
-    }
-    if (evt.which === NUMPAD_SUBSTRACT) {
-      this.onZoomOut();
-    }
-    if (evt.which === NUMPAD_ADD) {
-      this.onZoomIn();
+
+    if (isMacLike() == true) //Mac specific shortcuts -- zooming done differently on windows and linux
```

Comment:
I think we can keep these shortcuts (CMD/Ctrl+plus, CMD/Ctrl+minus, and the number + and -) for all systems right?